go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.6
-	github.com/rancher/gke-operator v1.0.1
+	github.com/rancher/gke-operator v1.0.2-rc2
```

Comment:
Note that this is the last 1.0.x version going into 2.5.x. We are backporting the 1.1.x version of gke-operator into 2.5.10+.